schemas/context/fileAttachment.schema.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://fdc3.finos.org/schemas/next/context/attachment.schema.json",
```

Comment:
the filename and this `$id` need to match, please change to: https://fdc3.finos.org/schemas/next/context/fileAttachment.schema.json